### PR TITLE
Skip loading istio configs from registry for remote clusters.

### PR DESCRIPTION
**Pull request by hhovsepy**
Fix for https://github.com/kiali/kiali/issues/6063

Skipping Registry usage for now for remote clusters because still the home cluster istiod is called when loading from registry.

Now the configs are not duplicated between clusters.
Tested also in primary remote mode when Istio CRDs are installed (https://raw.githubusercontent.com/istio/istio/master/manifests/charts/base/crds/crd-all.gen.yaml)

Screenshot from 2023-05-09 17-16-35

### CODE REVIEW

File: business/istio_config.go [hunk 1]
```diff
@@ -200,7 +200,8 @@ func (in *IstioConfigService) GetIstioConfigListPerCluster(ctx context.Context,
 	}
 
 	// Use the Istio Registry when AllNamespaces is present
-	if criteria.AllNamespaces && in.config.ExternalServices.Istio.IstioAPIEnabled {
+	// TODO use Istio Registry for Home cluster only now
+	if criteria.AllNamespaces && in.config.ExternalServices.Istio.IstioAPIEnabled && cluster == config.Get().KubernetesConfig.ClusterName {
```
[the remark this answers is not in the document]
Comment (hhovsepy) [reply business/istio_config.go:204]: tested on single and multiclusters istio_api_enabled set to false, istio configs are loaded fine, integration tests pass

**Comment by josunect** [comment]
Testing this in multi cluster, I have this errors: 

image

Also, it doesn't look to be fetching services from the remote cluster: 

image

In master: 

image


Also, I see some log errors: 

`2023-05-10T10:43:05+01:00 ERR Error fetching IstioConfigList per cluster west per namespace bookinfo: Kiali cache doesn't support [resourceType: VirtualService]
2023-05-10T10:43:05+01:00 ERR Unable to get services list from cluster: west. Err: Kiali cache doesn't support [resourceType: VirtualService]. Skipping`

This is tested with primary-remote support.

**Comment by josunect** [comment]
With the CRDs installed in the remote cluster, it is possible to see the Istio Configs: 

image

(If it is expected they are installed, probably the hack scripts should install them)
Tested also with istio_api_enabled to false

**Comment by hhovsepy** [comment]
@josunect @leandroberetta thank you for your comments, have reached them all

**Comment by hhovsepy** [comment]
ready for review now

**Review by josunect — APPROVED** [review]
LG

**Pull request merged**